src/android/plugin/google/maps/PluginTotalPaveTileLayer.java

```diff
@@ -50,7 +55,7 @@ public void create(final JSONArray args, final CallbackContext callbackContext)
                 .fadeIn(false)
             );
 
-            String id = "totalPaveTileLayer_" + hashCode;
+            String id = "totalpavetilelayer_" + hashCode;
             this.pluginMap.objects.put(id, overlay);
```

Comment:
Apparently cordova has some internal code where this value ends up being the lower case variant when obtaining it from javascript getId.